src/pages/scene-viewer.tsx

```diff
@@ -48,6 +61,17 @@ export default function SceneViewer(): JSX.Element {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [credentials]);
 
+  React.useEffect(() => {
+    if (viewer.ref.current && !viewId) {
+      viewer.ref.current.addEventListener("sceneReady", async () => {
+        if (viewer.ref.current) {
+          const scene = await viewer.ref.current.scene();
+          setViewId(scene.sceneViewId);
+        }
+      });
+    }
+  }, [viewer.ref, viewer.ref.current, viewId]); // eslint-disable-line
```

Comment:
Instead of a blanket ignore, ideally we'd just ignore specific errors, `// eslint-disable-next-line react-hooks/exhaustive-deps`. There should be a context menu action for it (if not, try installing the ESLint extension in VS Code)